src/react/account/AccountUserSecretKeyModal.tsx

```diff
@@ -106,9 +116,13 @@ const modalBody = (key: AccountKey | null) => {
       >
         <T.Body>
           <T.Row>
-            <T.Key> User </T.Key>
+            <T.Key> Account Name </T.Key>
             <T.Value> {key.accountName} </T.Value>
```

Comment:
I think we name it `accountName` here is misleading, we should rename it to userName
https://github.com/scality/zenko-ui/blob/development/1.5/src/react/account/AccountUserSecretKeyModal.tsx#L28